doc/guide.rst

```diff
@@ -0,0 +1,179 @@
+
+            # Remove continue option from the control. There is no need for that
+            # when we have only one screen.
+            prompt.remove_option('c')
+
+            return prompt
+
+When we are able to present our content to a user we want to have the possibility to process
+user input. For this purpose there is the :meth:`UIScreen.input` method. Input from a user is
+passed to this method and the screen may process it, discard it or return it for further
+option processing.
+
+If input processing is not required and screen should be used only for displaying
+information to a user then the :attr:`UIScreen.input_required` property should be set to `False`,
+however, in this case you need to :meth:`close <UIScreen.close>`, :meth:`redraw <UIScreen.redraw>`
+or :ref:`push <screen_handling_label>` a new screen manually. That can be done for example in the
+:meth:`UIScreen.show_all` method.
+
+.. code-block:: python
+    :dedent: 0
+
+        def input(self, args, key):
+            """Process input from user and catch numbers with '/' symbol."""
+
+            # Test if user passed valid input for divider.
+            # This will basically take number + number and nothing else and only positive numbers.
+            groups = re.match(r'(\d+) *\/ *(\d+)$', key)
+            if groups:
+                num1 = int(groups[1])
+                num2 = int(groups[2])
+
+                # Dividing by zero is not valid so we won't accept this input from the user. New
+                # input is then required from the user.
+                if num2 == 0:
+                    return InputState.DISCARDED
+
+                self._message = int(num1 / num2)
+
+                # Because this input is processed we need to show this screen (show the result)
+                # manually by calling the redraw() method. This will call refresh so our new result
+                # will be processed inside of the refresh() method.
+                self.redraw()
+                return InputState.PROCESSED
+            else:
+                # Not input for our screen, try other default inputs. This will result in the
+                # same state as DISCARDED when no default option is used.
+                return key
+
+
+.. py:currentmodule:: simpleline
+
+Our screen is finished so next we need to use it in our application. To run an application the
+:class:`App` static class must be used.
+
+This class will initialize event loop and
+scheduler by the :meth:`App.initialize` method. When the application is initialized we need to
+pass our screen to the screen stack (you can pass multiple screens but we have only one here).
+To pass a screen to the screen stack in the scheduler initialized
+by :class:`App` we will use :class:`ScreenHandler <simpleline.render.screen_handler.ScreenHandler>`.
+To get deeper introduction how to schedule screens look on the :ref:`screen_handling_label` section.
```

Comment:
`To get deeper introduction how to schedule screens look on the <ref> section.` -> `For further explanation on scheduling screens, refer to the <ref> section.`